drivers/src/pci/root.rs

```diff
@@ -78,7 +78,7 @@ impl PcieRoot {
             // Any overlaps will be caught by `add_mmio_region` and the error will be propagated.
             mem_map
                 .add_mmio_region(
-                    DeviceMemType::PciRoot,
+                    DeviceMemType::PciConfig,
```

Comment:
Not related, but commit message needs a small tweak: "...since we're using **_it_** to...".

Reply:
Done.